blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/rest/PipelineImpl.java

```diff
@@ -164,4 +177,28 @@ protected static String getRecursivePathFromFullName(BluePipeline pipeline){
         return pipelinePath.toString();
     }
 
+    @Extension(ordinal = 1)
+    public static class PipelineFactoryImpl extends BluePipelineFactory {
```

Comment:
Well, this is inner class, implementation name, does it matter to be so specific about naming? I would worry about it if it was an interface or contract like what goes in blueocean-rest module.